### PR TITLE
CVE-2011-1413

### PR DESCRIPTION
**Pull request by JakeBash**
Update the YAML file for the assigned CVE and answered the questions within the file

### CODE REVIEW

File: cves/CVE-2011-1413.yml [hunk 1]
```diff
@@ -88,7 +78,6 @@ upvotes_instructions: |
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
 upvotes:
```
Comment (heartemoji) [inline cves/CVE-2011-1413.yml]: 4 upvotes

File: cves/CVE-2011-1413.yml [hunk 2]
```diff
@@ -36,11 +32,11 @@ description_instructions: |
...
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: "(Linux Only) If a malicious user remotely constructed a realtively long string
+description: |
+  (Linux Only) If a malicious user remotely constructed a relatively long string
   and used the javascript alert function, or used it with an input that could be 
   displayed in an alert, it could crash the display server and lock up the 
-  chrome windows on the target device, requiring a reboot (Denial of Service)"
-
+  chrome windows on the target device, requiring a reboot (Denial of Service)
```
[the remark this answers is not in the document]
Comment (JakeBash) [reply cves/CVE-2011-1413.yml]: Now that you mention it, I agree. I'll be sure to make some changes here with the next commit.

File: cves/CVE-2011-1413.yml [hunk 3]
```diff
@@ -152,7 +139,6 @@ interesting_commits:
     note: They worked on the design of the windows and dialogs
```
[the remark this answers is not in the document]
Comment (JakeBash) [reply cves/CVE-2011-1413.yml]: I'll dig for some juicier details about the commit that give a bit more context for the reader. Thanks for the feedback!

File: cves/CVE-2011-1413.yml (hunk 1 above)
Comment (JakeBash) [reply cves/CVE-2011-1413.yml]: Thanks for the upvotes! I'll be sure to give you some shortly!

**Pull request merged**